### ISSUE TITLE
Update email copy for inviting a user to create a shop

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#2804]
We need to update the copy for the email that goes out when a marketplace admin invites a new user to create/manage a new shop on their marketplace platform.

This issue can serve as the base for the discussion as to what this email should contain.

Here is the current email:
![image](https://user-images.githubusercontent.com/1203639/30175180-e285ba46-93ba-11e7-9088-8f1aaa154eea.png)

At a minimum, the updated copy should clarify:
- Who/which marketplace is doing the inviting, right now you're being invited by the new shop that you're asked to create. This doesn't make sense.
- What you'll need to do to accept this invitation
- That this is _not_ marketing email and you've been invited directly by the marketplace manager.

We can - and perhaps should - include the new name of the shop in the form when the invite is created rather than appending a "1" or leaving off the shop name?

Other thoughts on this?

**Comment by kieckhafer** [comment]
I think there are 6 places where we need to decide which shop information is being applied. I think all 6 should use the main marketplace shop, instead of the newly created / invited shop. 4 & 5 might be removed if we get new copy, but the rest will probably stay.

<img width="985" alt="airmail" src="https://user-images.githubusercontent.com/4482263/30338208-a6797f04-979f-11e7-9b31-3628fb7b31f8.png">

Thoughts?

**Comment by spencern** [comment]
I'm for updating all of them to use the main marketplace. Possible that as we update the invite flow, we might be able to use the `ShopName` they are invited to manage somewhere (`Admin has invited you to manage a shop` feels so bleh)

**Comment by rymorgan** [comment]
@spencern @kieckhafer -- Here's my take. It's a bit generic b/c I would imagine most marketplaces would want to add their own style to it. 

Subject line: $Marketplace has invited you to join as a shop

Email:
Hi, $username 

You’ve been invited by $Marketplace to set up a shop. Click below to configure your shop and start adding products. 

Get Started CTA

—————
You received this email because $Marketplace invited you to manage a shop. Questions and suggestions? Email us at info@$marketplace.com

**Comment by spencern** [comment]
I think if we get the language consistent it's definitely better than what we have now.
I think I lean what if we used "create a store"? Of the three you've got I like "manage a store best" but not sure that fully conveys the idea.

![image](https://user-images.githubusercontent.com/1203639/30412038-1cef4c1c-98d2-11e7-8e40-87348f2c45eb.png)

**Comment by rymorgan** [comment]
@spencern Fair point. 

Subject line: $Marketplace has invited you to manage a store

Email:
Hi, $Merchant

You’ve been invited by $Marketplace to manage a store. Click below to configure your store and start adding products.

Get Started CTA

—————
You received this email because $Marketplace invited you to manage a store. Questions and suggestions? Email us at $marketplaceEmail.

**Comment by impactmass** [comment]
I'm currently updating the owner email based on the final copy on the thread here (#2898). 
I think we should also update the copy for inviting a user to a group as well. 

It currently looks like the first screenshot above:
```
Subject line: You have been invited to join {{ShopName}}

Email:
Hi, {{InvitedUser}}

{{currentUserName}} has invited you to manage a shop.

Get Started CTA

—————
You received this email because a user of {{shopName}} invited you to manage a shop. Questions or suggestions? Email us at {{contactEmail}}
```

Here's a draft I have:
```
Subject line: You have been invited to {{groupName}} in {{shopName}}

Email:
Hi, {{InvitedUser}}

{{currentUserName}} has invited you to {{groupName}} in {{shopName}}

Get Started CTA

—————
You received this email because a user of {{shopName}} invited you to a group. Questions or suggestions? Email us at {{contactEmail}}
```
thoughts? cc @rymorgan @spencern

**Comment by spencern** [comment]
Hey @impactmass 
I agree that we should have an email like this. I don't see anything wrong with your copy here, but will defer to @rymorgan on exact copy.

**Comment by rymorgan** [comment]
I'm good with that copy update.

**Comment by impactmass** [comment]
closed by #2898
